src/xo-app/index.js

```diff
@@ -37,6 +38,18 @@ import Vm from './vm'
 export default class XoApp extends Component {
   componentDidMount () {
     this.refs.bodyWrapper.style.minHeight = this.refs.menu.getWrappedInstance().height + 'px'
+    const previousDisclaimer = cookies.get('previousDisclaimer')
```

Comment:
Where is your mind about handling build differences between various appliances ? Depending on the way we do that, we can make a final choice on where and how we should write this piece of code.